CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 - `nflow-engine`
   - BREAKING CHANGE: Remove `WorkflowDefinition`, workflow definitions should extend `AbstractWorkflowDefinition` instead.
   - BREAKING CHANGE: Remove deprecated `WorkflowState.isRetryAllowed`, set exception analyzer for workflow definition instead (if needed).
+  - BREAKING CHANGE: Change transition delay fields data types in `WorkflowSettings` and remove unused `immediateTransitionDelay` setting.
+- `nflow-rest-api`
+  - BREAKING CHANGE: Remove unused `ListWorkflowDefinitionResponse.TransitionDelays.immediate` field.
```

Comment:
Maybe specify that this is REST api change? No-one normally uses that class directly and it might not be obvious from the name.

Reply:
It is under `nflow-rest-api` bullet, should be obvious from there?